template/test/renderer/Store.spec.js

```diff
@@ -0,0 +1,24 @@
+import Store from 'stores';
+
+describe('{{pascalcase name}}Store [Store]', () => {
+  beforeEach(function() {
```

Comment:
these tests are just a copy of the Store.spec.js file, right? Might be confusing to users of this plugin. Can we think of a test that actually needs to be executed in the renderer process instead?

Reply:
@rueckstiess I agree, and given more time I could probably come up with a more contrived example to better demonstrate, but the important thing was simply having something there to test to demonstrate the Karma-Electron renderer tests run and pass.

If you're ok with leaving this as is for now, I can address this comment with a better example as part of another PR when I have more time and fewer higher priority items to get through. How does that sound?

Reply:
ok, let's leave as is. But could you add a comment on the top of the file, something like this (feel free to change the wording).

```
/*
 * Place tests that must run in a renderer context inside Electron here. 
 * 
 * Note: The tests below are just a copy of the store unit tests as an example. 
 * More complex plugins will require actual renderer/integration tests to be
 * executed here.
 */

Reply:
@rueckstiess Yeah that's a good idea, I'll add the comment and then return to this in another PR.